sys/include/net/ng_ipv6/nc.h

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+/**
+ * @defgroup    net_ng_ipv6_nc  IPv6 neighbor cache
+ * @ingroup     net_ng_ipv6
+ * @brief       Translates IPv6 addresses to link layer addresses.
+ * @{
+ *
+ * @file
+ * @brief       Neighbor cache definitions.
+ *
+ * @author      Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+
+#ifndef NG_IPV6_NC_H_
+#define NG_IPV6_NC_H_
+
+#include <stdint.h>
+
+#include "kernel_types.h"
+#include "net/ng_ipv6/addr.h"
+#include "net/ng_netif.h"
+#include "net/ng_pktqueue.h"
+#include "timex.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   The size of the neighbor cache
+ */
+#define NG_IPV6_NC_SIZE             (NG_NETIF_NUMOF * 8)
+
+/**
+ * @brief   The maximum size of a link layer address
+ */
+#define NG_IPV6_L2_ADDR_MAX         (8)
+
+/**
+ * @{
+ * @name Flag definitions for ng_ipv6_nc_t
+ */
+/**
+ * @{
+ * @brief   States of a neighbor cache entry.
+ *
+ * @see <a href="http://tools.ietf.org/html/rfc4861#section-7.3.2">
+ *          RFC 4861, section 7.3.2
+ *      </a>
+ */
+#define NG_IPV6_NC_STATE_MASK           (0x07)
+#define NG_IPV6_NC_STATE_POS            (0)
+
+#define NG_IPV6_NC_STATE_UNMANAGED      (0x00)  /**< The entry is not manage by NDP */
+
+/**
+ * @brief The entry is unreachable
```

Comment:
no, it documents the `NG_IPV6_NC_STATE_UNREACHABLE` macro, which is exactly below this comment block.